src/common-components/data/service.js

```diff
@@ -18,10 +18,8 @@ export async function getThirdPartyAuthContext(urlParams) {
       throw (e);
     });
   return {
-    fieldDescriptions: data.registrationFields || data.registration_fields || {},
-    optionalFields: data.optionalFields || data.optional_fields || {},
-    thirdPartyAuthContext: data.contextData || camelCaseObject(
-      convertKeyNames(data.context_data, { fullname: 'name' }),
-    ),
+    fieldDescriptions: data.registrationFields,
+    optionalFields: data.optionalFields,
+    thirdPartyAuthContext: data.contextData,
```

Comment:
Have you tested this? This will probably break the code when the MFEContext API is in the "fetching" state or has failed. We will need the default values.

Reply:
MFEContext API will have these particular keys included with default values. However, I have not yet tested the scenario where the API fails. I have just checked the test results. I will investigate this further and create a test scenario to verify the behavior in case of an API failure.

Reply:
I have included a condition in the API response to handle default values.